src-plugins/itkFilters/itkFiltersAddProcess.cpp

```diff
@@ -11,38 +11,37 @@
 
 =========================================================================*/
 
-#include <itkFiltersAddProcess.h>
+#include "itkFiltersAddProcess.h"
 
 #include <dtkCore/dtkAbstractProcessFactory.h>
-#include <medAbstractDataFactory.h>
 
-#include <medMetaDataKeys.h>
+#include <itkAddImageFilter.h>
+#include <itkImage.h>
 
-#include <itkFiltersAddProcess_p.h>
+#include <medUtilities.h>
+
+class itkFiltersAddProcessPrivate
+{
+public:
+    double addValue;
+};
 
 //-------------------------------------------------------------------------------------------
 
 itkFiltersAddProcess::itkFiltersAddProcess(itkFiltersAddProcess *parent) 
-    : itkFiltersProcessBase(*new itkFiltersAddProcessPrivate(this), parent)
-{
-    DTK_D(itkFiltersAddProcess);
-    
-    d->filter = this;
-    d->output = NULL;
-    
-    d->description = tr("ITK add constant filter");
-}
-
+    : itkFiltersProcessBase(), d(new itkFiltersAddProcessPrivate)
+{  
+    filter = this;
+    descriptionText = tr("ITK add constant filter");
 
-itkFiltersAddProcess::itkFiltersAddProcess(const itkFiltersAddProcess& other) 
-    : itkFiltersProcessBase(*new itkFiltersAddProcessPrivate(*other.d_func()), other)
-{
+    d->addValue = 100;
```

Comment:
As a reminder, before this PR, we did not enter these constructors, the variables were never initialized. That's why in pipelines for instance, if we did not use "setParameter(...)" to init these variables, we had bugs

Reply:
We shouldn't initialise variables to arbitrary values. I would set most of the filter parameters to 1, 0 or the maximum theoretical value, depending on the case (1 for addition, subtraction, multiplication, division, size threshold, morphological radius etc.).

Reply:
these variables are not arbitrary values, these are the default numbers in the toolboxes

Reply:
Wherever the default values come from, they are still arbitrary (why 100?). In any case if this is supposed to be the same value as the toolbox then it might be a good idea to enforce that by putting a public constant in the ```itkFiltersAddProcess``` class and having both this class and ```itkFiltersToolBox``` take their value from that.

Reply:
If it is set this way to have the same default value as the toolbox I can understand that the value is set to something more "usable" (100 is probably closer to what the user wants than 1, so they would have less "spinning" to do on the GUI), but it is just strange to have 100 here, 200 for the threshold etc. This is what I mean by arbitrary, because there is no obvious pattern between the values.